src/regions/ca/info/faq/41_too-late-to-wash-hands.md

```diff
@@ -0,0 +1,7 @@
+## Is it too late, at some point, to wash your hands?
+
+It's never too late to wash your hands and remove viruses and bacteria. The longer you wait to wash your hands and the more surfaces or parts of your face you reach in that time, the more likely you are to contaminate others or yourself.
+
+So wash your hands carefully for at least 20 seconds on a regular basis. Here is how to wash your hand proprely : https://www.canada.ca/en/public-health/services/publications/diseases-conditions/reduce-spread-covid-19-wash-your-hands.html
```

Comment:
Can we change this so it is not shown as a link in the page?

Reply:
Fixed!